src/packages/migrate/src/commands/SchemaPush.ts

```diff
@@ -0,0 +1,132 @@
+import { arg, Command, format, HelpError, isError } from '@prisma/sdk'
+import chalk from 'chalk'
+import { Migrate, PushOptions } from '../Migrate'
+import { ensureDatabaseExists } from '../utils/ensureDatabaseExists'
+import { ExperimentalFlagError } from '../utils/experimental'
+import { formatms } from '../utils/formatms'
+
+export class SchemaPush implements Command {
+  public static new(): SchemaPush {
+    return new SchemaPush()
+  }
+
+  // static help template
+  private static help = format(`
+    Push the state from your schema.prisma to your database
+
+    ${chalk.bold.yellow('WARNING')} ${chalk.bold(
+    "Prisma's schema push functionality is currently in an experimental state.",
+  )}
+    ${chalk.dim(
+      'When using any of the commands below you need to explicitly opt-in via the --experimental flag.',
+    )}
+
+    ${chalk.bold('Usage')}
+
+      ${chalk.dim('$')} prisma schema push --experimental
+
+    ${chalk.bold('Options')}
+
+      --force           Skip interactive approval before migrating
+      -c, --create-db   Create the database in case it doesn't exist
+      -h, --help        Displays this help message
+
+    ${chalk.bold('Examples')}
+
+      Push the local schema state to the database
+      ${chalk.dim('$')} prisma schema push --experimental
+
+      Push the local schema state to the database
+      ${chalk.dim('$')} prisma schema push --experimental
```

Comment:
Is the repetition here intentional?

Reply:
I wanted to add an example with --force, will fix!